packages/fast-usdc/test/cli/transfer.test.ts

```diff
@@ -97,7 +120,7 @@ test('Transfer registers the noble forwarding account if it does not exist', asy
     { signer: signerMock.signer, address: nobleSignerAddress },
     mockEthProvider.provider,
   );
-
+  console.log(out.getLogOut());
```

Comment:
our tests are pretty noisy as it is

Reply:
Removed thanks, accidentally left this in from debugging